src/main/java/frc/robot/Constants.java

```diff
@@ -46,10 +46,25 @@
         public static final class LauncherConstants {
             public static final int kRightLauncherMotorPort = 13;
             public static final int kLeftLauncherMotorPort = 14;
+
+            public static final double kEncoderSpeedPerPulse = 1; // ADD CONVERSION
```

Comment:
No conversion necessary, since the SPARKMAX outputs RPMs by default, and we're not trying to turn this into a linear speed.